tvision/classes/tdirlist.cc

```diff
@@ -204,7 +203,7 @@ void TDirListBox::showDrives( TDirCollection *dirs )
 {
     Boolean isFirst = True;
     char oldc[5];
-    strcpy( oldc, "0:"DIRSEPARATOR_ );
+    strcpy( oldc, "0:" DIRSEPARATOR_ );
```

Comment:
This space between "0:" and DIRSEPARATOR_ is required.  Without it, the MS compiler generates the following errors:

`../classes/tdirlist.cc(206): error C3688: invalid literal suffix 'DIRSEPARATOR_'; literal operator or literal operator template 'operator ""DIRSEPARATOR_' not found

../classes/tdirlist.cc(206): error C2660: 'strcpy': function does not take 1 arguments

D:\Windows Kits\10\include\10.0.19041.0\ucrt\string.h(130): note: see declaration of 'strcpy'
`